app/src/views/private/components/revision-item.vue

```diff
@@ -30,7 +30,7 @@ const revisionCount = computed(() => {
 const headerMessage = computed(() => {
 	switch (props.revision.activity.action.toLowerCase()) {
 		case 'create':
-			return t('revision_delta_updated', revisionCount.value);
+			return t('revision_delta_created', revisionCount.value);
```

Comment:
Let’s remove the parameter here since there is none for the `revision_delta_created` message.

```suggestion
			return t('revision_delta_created');
```